woocommerce/woocommerce-ios
# Storage: Add new attribute `wasEcommerceTrial` to Site entity.

Part of #10298 


## Description

This PR updates the Site entity in Storage with the new attribute `wasEcommerceTrial`.

## Testing instructions

Build and run the app after running it from another branch - the app should still run and doesn't crash. 
Also, make sure that CI still passes.

## Screenshots

N/A

---
